testutil/e2e/sdk/tests/emergency_mode_fetch_err.ts

```diff
@@ -32,22 +32,23 @@ async function main() {
 
             // Validate chainID
             if (chainID !== "lava") {
-                throw new Error(" ERR [tendermintrpc_chainid_fetch] Chain ID is not equal to lava");
+                throw new Error(" ERR [emergency_mode_fetch_err] Chain ID is not equal to lava");
             } else {
-                console.log(i, "[tendermintrpc_chainid_fetch] Success: Fetching Lava chain ID using tendermintrpc passed. Chain ID correctly matches 'lava'");
+                console.log(i, "[emergency_mode_fetch_err] Success: Fetching Lava chain ID using tendermintrpc passed. Chain ID correctly matches 'lava'");
             }
         } catch (error) {
-            throw new Error(` ERR ${i} [tendermintrpc_chainid_fetch] failed sending relay tendermint test: ${error.message}`);
+            throw new Error(` ERR ${i} [emergency_mode_fetch_err] failed sending relay tendermint test: ${error.message}`);
         }
     }
 }
 
 (async () => {
+    console.log("Start")
```

Comment:
Should it align with the rest of the tests? 
Maybe add this line to the others too